### PR TITLE
Add Veikk A30 V2

### PR DESCRIPTION
**Pull request by SnowLire**
Still waiting on the guy who asked for support to come back and test it, although I suspect it's the same as the Veikk S640 V2 and only needed a parser change.

Conversation: https://discord.com/channels/615607687467761684/615611007951306863/965638577394425926

### CODE REVIEW

**Comment by InfinityGhost** [comment]
Untested configurations will not be merged. Marking as draft until this is verified.

**Comment by jamesbt365** [comment]
Any reason this isn't in the same Configuration as the A30? There's no conflict between them.

File: OpenTabletDriver.Configurations/Configurations/VEIKK/A30 V2.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      "InputReportLength": 13,
+      "OutputReportLength": 9,
+      "ReportParser": "OpenTabletDriver.Configurations.Parsers.Veikk.VeikkReportParser",
+      "FeatureInitReport": null,
+      "OutputInitReport": [
+        "CQEE"
+      ],
+      "DeviceStrings": {},
+      "InitializationStrings": []
+    }
```
Comment (X9VoiD) [inline OpenTabletDriver.Configurations/Configurations/VEIKK/A30 V2.json:35]: Move inside `VEIKK A30` configuration as this has similar specifications to that.

**Review by X9VoiD — CHANGES_REQUESTED** [review]

File: OpenTabletDriver.Configurations/Configurations/VEIKK/A30 V2.json (hunk 1 above)
Comment (SnowLire) [reply OpenTabletDriver.Configurations/Configurations/VEIKK/A30 V2.json:35]: Did it this way to stay consistent with the Veikk S640 and S640 V2 being 2 different configs despite having only the `InputReportLength` changed.

Comment (Kuuuube) [reply OpenTabletDriver.Configurations/Configurations/VEIKK/A30 V2.json:35]: The Veikk A30 V2 and Veikk A30 are both product names and how the tablets are referred to by Veikk. They should not be combined.

Comment (jamesbt365) [reply OpenTabletDriver.Configurations/Configurations/VEIKK/A30 V2.json:35]: https://discord.com/channels/615607687467761684/628651971267788800/977920269509357641

Comment (X9VoiD) [reply OpenTabletDriver.Configurations/Configurations/VEIKK/A30 V2.json:35]: Aight, will mark as resolved.

**Review by X9VoiD — APPROVED** [review]

**Review by X9VoiD — APPROVED** [review]

**Review by jamesbt365 — APPROVED** [review]

**Review by jamesbt365 — APPROVED** [review]

**Review by jamesbt365 — APPROVED** [review]

**Review by jamesbt365 — CHANGES_REQUESTED** [review]

**Comment by jamesbt365** [comment]
Verification: https://discord.com/channels/615607687467761684/615611007951306863/978675579551363123

**Comment by jamesbt365** [comment]
hold off, some aux fuckery

File: TABLETS.md [hunk 2]
```diff
@@ -180,6 +180,7 @@
 | XP-Pen Star 06C               |  Missing Features | Wheel is not yet supported.
 | Wacom ISD-V4                  |      Broken       | [#394](https://github.com/OpenTabletDriver/OpenTabletDriver/issues/394)
 | Huion G10T                    |     Untested      |
+| Veikk A30 V2                  |     Untested      |
```
[the remark this answers is not in the document]
Comment (SnowLire) [reply TABLETS.md]: Resolved

**Review by jamesbt365 — APPROVED** [review]

**Pull request merged**